lib/puppet/parser/functions/ip_to_cron.rb

```diff
@@ -0,0 +1,55 @@
+#
+# ip_to_cron.rb
+#
+# Copyright 2016 Vox Pupuli
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+module Puppet::Parser::Functions
+  newfunction(:ip_to_cron, :type => :rvalue, :doc => <<-EOS
+Returns an array of valid cron times given an optional runinterval
```

Comment:
IMO, runinterval should be required. And the seed provided to fqdn_rand should also be a parameter too right?